chia/rpc/wallet_rpc_api.py

```diff
@@ -1185,14 +1185,30 @@ async def create_signed_transaction(self, request, hold_lock=True):
         if "coins" in request and len(request["coins"]) > 0:
             coins = set([Coin.from_json_dict(coin_json) for coin_json in request["coins"]])
 
+        coin_announcements: Set[bytes32] = None
```

Comment:
I think you will have to change this to Optional[Set[bytes32]] because None is not a Set. 
Or initialize to set()